includes/definitions/zywall.yaml

```diff
@@ -6,7 +6,10 @@ over:
     - { graph: device_bits, text: Traffic }
 icon: zyxel
 discovery:
-    - sysObjectId:
-        - .1.3.6.1.4.1.890.1.6
-        - .1.3.6.1.4.1.890.1.15
+    -
+      sysObjectId:
+          - .1.3.6.1.4.1.890.1.6
+          - .1.3.6.1.4.1.890.1.15
```

Comment:
My thought was this:
```
   -
      sysObjectId:
          - .1.3.6.1.4.1.890.1.6
          - .1.3.6.1.4.1.890.1.15
      sysDescr_regex:
          - '/^ZyWALL/
```

I'm not a fan of negative matches, they make a big mess.